src/lib/pickles/plonk_checks/plonk_checks.ml

```diff
@@ -167,11 +176,19 @@ let expand_feature_flags (type boolean)
   in
   let table_width_1 =
     (* RangeCheck, ForeignFieldMul have max_joint_size = 2 *)
-    lazy (B.any [ Lazy.force table_width_2; range_check; foreign_field_mul ])
+    lazy
+      (B.any
+         [ Lazy.force table_width_2
```

Comment:
Why is above `table_width_1` if this is `table_width_2`?

Reply:
It might be clearer to rename this to `table_width_at_least_1`; any higher table width should cascade down to cause this to also be true.